pkg/flowkit/config/json/account.go

```diff
@@ -94,11 +94,23 @@ func transformSimpleToConfig(accountName string, a simpleAccount) (*config.Accou
 // tryReplaceEnv checks if value matches env regex, if it does it check whether the value was set in env,
 // if not set then it errors, otherwise it replaces the value with set env variable, and also returns the original key.
 func tryReplaceEnv(value string) (replaced string, original string, err error) {
-	if envPresent, _ := regexp.MatchString("^\\$\\w+", value); !envPresent {
+	envRegex, err := regexp.Compile(`^\$[{]?(\w+)[}]?`)
```

Comment:
maybe `^\$((\w+)|\{(\w+)\})` to prevent partial match

Reply:
good catch